packages/shared/src/hooks/useLogin.ts

```diff
@@ -55,18 +54,13 @@ const useLogin = ({
   onSuccessfulLogin,
   queryEnabled = true,
   queryParams = {},
-  enableSessionVerification = false,
```

Comment:
As we can see, the session is optional only from this point, hence, I think it would be better to have it just a prop since it is not necessary on all login flows.

Instead, the security page needs the session all the time, so it would be better to instantiate it there.